challenge_2/python/mical/main.py

```diff
@@ -0,0 +1,12 @@
+#02/01/2017
+
+def noRepeats(list):
+    for x in list:
+        amount = list.count(x)
```

Comment:
Isnt `list` a python reserved keyword? Regardless... Naming a variable after a data structure isn't the best habit. I would advise to change it to something more accurate! Otherwise, good job!